applications/asset_tracker/src/main.c

```diff
@@ -139,7 +140,8 @@ enum error_type {
 	ERROR_CLOUD,
 	ERROR_BSD_RECOVERABLE,
 	ERROR_BSD_IRRECOVERABLE,
-	ERROR_LTE_LC
+	ERROR_LTE_LC,
+	ERROR_THREAD
```

Comment:
Why ERROR_THREAD? I think that ERROR_SYSTEM or ERROR_SYSTEM_FAULT would be more descriptive.

Reply:
Changed to ERROR_SYSTEM_FAULT.